app/package.json

```diff
@@ -10,20 +10,28 @@
     "@mui/icons-material": "^5.15.14",
     "@mui/material": "^5.15.14",
     "@mui/material-nextjs": "^7.0.2",
+    "@types/dompurify": "^3.0.5",
     "@types/js-cookie": "^3.0.6",
+    "@types/jsdom": "^21.1.7",
+    "@types/tar-stream": "^3.1.4",
```

Comment:
move all `@types` deps to `devDependencies` since these are compile-time only; adds unnecessary weight to prod builds served to user